src/App.js

```diff
@@ -0,0 +1,82 @@
+import "macro-css";
+import './index.scss';
+import { MainTemperatureBlock } from "./components/DateTemperature/MainTemperatureBlock/index.js";
+import { SearchToolbar } from "./components/SearchToolbar";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+import { DayTemperatureBlock } from "./components/DayTemperature";
+
+
+const indexOfItem = [2, 5, 8, 11, 14, 17, 20, 23]
```

Comment:
ok, i think will be better to move it to utils/constants/temperature-indexes-template.js and import it as TEMPERATURE_INDEXES_TEMPLATE